Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/Contoso.Forms.Puppet.UWP.csproj

```diff
@@ -12,76 +12,80 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10586.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.14393.0</TargetPlatformMinVersion>
```

Comment:
You're saying we should change the minimum platform version of the app to be the same as the minimum platform version of the sdk?